src/test/java/com/limechain/storage/crypto/KeyStoreTest.java

```diff
@@ -0,0 +1,36 @@
+package com.limechain.storage.crypto;
+
+import com.limechain.chain.Chain;
+import com.limechain.storage.DBInitializer;
+import com.limechain.storage.DBRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class KeyStoreTest {
+
+    private DBRepository dbRepository;
+
+    private KeyStore keyStore;
+
+    @BeforeEach
+    public void setup() {
+        dbRepository = DBInitializer.initialize("./test", Chain.WESTEND, false);
```

Comment:
Do we need to initialize a test DB, can't we just mock the repository?

Reply:
We need to test the functionality.